CHANGES.rst

```diff
@@ -15,6 +15,9 @@ astropy.convolution
 
 astropy.coordinates
 ^^^^^^^^^^^^^^^^^^^
+- Added the ``direction`` feature to the angle initialization.
```

Comment:
I think an example and mentioning the word "cardinal direction" would help here.

Something along the lines of "Angle parsing now supports cardinal directions, e.g. `17°53'27"W`"